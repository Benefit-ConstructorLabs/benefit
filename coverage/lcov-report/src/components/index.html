<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> src/components
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">23.57% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>74/314</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">14.94% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>23/154</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">16.82% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>18/107</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">24.1% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>74/307</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file high" data-value="AnimationWrapper.js"><a href="AnimationWrapper.js.html">AnimationWrapper.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	</tr>

<tr>
	<td class="file medium" data-value="App.js"><a href="App.js.html">App.js</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	</tr>

<tr>
	<td class="file high" data-value="AppFooter.js"><a href="AppFooter.js.html">AppFooter.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	</tr>

<tr>
	<td class="file low" data-value="AppHeader.js"><a href="AppHeader.js.html">AppHeader.js</a></td>
	<td data-value="8.7" class="pic low"><div class="chart"><div class="cover-fill" style="width: 8%;"></div><div class="cover-empty" style="width:92%;"></div></div></td>
	<td data-value="8.7" class="pct low">8.7%</td>
	<td data-value="23" class="abs low">2/23</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="22" class="abs low">0/22</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="8.7" class="pct low">8.7%</td>
	<td data-value="23" class="abs low">2/23</td>
	</tr>

<tr>
	<td class="file low" data-value="AppWrapper.js"><a href="AppWrapper.js.html">AppWrapper.js</a></td>
	<td data-value="20" class="pic low"><div class="chart"><div class="cover-fill" style="width: 20%;"></div><div class="cover-empty" style="width:80%;"></div></div></td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	</tr>

<tr>
	<td class="file low" data-value="CardSection.js"><a href="CardSection.js.html">CardSection.js</a></td>
	<td data-value="0" class="pic low"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	</tr>

<tr>
	<td class="file low" data-value="Dashboard.js"><a href="Dashboard.js.html">Dashboard.js</a></td>
	<td data-value="3.13" class="pic low"><div class="chart"><div class="cover-fill" style="width: 3%;"></div><div class="cover-empty" style="width:97%;"></div></div></td>
	<td data-value="3.13" class="pct low">3.13%</td>
	<td data-value="32" class="abs low">1/32</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="14" class="abs low">0/14</td>
	<td data-value="3.57" class="pct low">3.57%</td>
	<td data-value="28" class="abs low">1/28</td>
	</tr>

<tr>
	<td class="file low" data-value="DonationForm.js"><a href="DonationForm.js.html">DonationForm.js</a></td>
	<td data-value="36.36" class="pic low"><div class="chart"><div class="cover-fill" style="width: 36%;"></div><div class="cover-empty" style="width:64%;"></div></div></td>
	<td data-value="36.36" class="pct low">36.36%</td>
	<td data-value="11" class="abs low">4/11</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="14.29" class="pct low">14.29%</td>
	<td data-value="7" class="abs low">1/7</td>
	<td data-value="36.36" class="pct low">36.36%</td>
	<td data-value="11" class="abs low">4/11</td>
	</tr>

<tr>
	<td class="file high" data-value="DonationHeader.js"><a href="DonationHeader.js.html">DonationHeader.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	</tr>

<tr>
	<td class="file medium" data-value="DonationPayment.js"><a href="DonationPayment.js.html">DonationPayment.js</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	</tr>

<tr>
	<td class="file low" data-value="Donor.js"><a href="Donor.js.html">Donor.js</a></td>
	<td data-value="20" class="pic low"><div class="chart"><div class="cover-fill" style="width: 20%;"></div><div class="cover-empty" style="width:80%;"></div></div></td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	</tr>

<tr>
	<td class="file low" data-value="DonorBalance.js"><a href="DonorBalance.js.html">DonorBalance.js</a></td>
	<td data-value="28.57" class="pic low"><div class="chart"><div class="cover-fill" style="width: 28%;"></div><div class="cover-empty" style="width:72%;"></div></div></td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="7" class="abs low">2/7</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	</tr>

<tr>
	<td class="file medium" data-value="DonorProfile.js"><a href="DonorProfile.js.html">DonorProfile.js</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	</tr>

<tr>
	<td class="file medium" data-value="HomePage.js"><a href="HomePage.js.html">HomePage.js</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	</tr>

<tr>
	<td class="file low" data-value="Login.js"><a href="Login.js.html">Login.js</a></td>
	<td data-value="42.86" class="pic low"><div class="chart"><div class="cover-fill" style="width: 42%;"></div><div class="cover-empty" style="width:58%;"></div></div></td>
	<td data-value="42.86" class="pct low">42.86%</td>
	<td data-value="7" class="abs low">3/7</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	<td data-value="42.86" class="pct low">42.86%</td>
	<td data-value="7" class="abs low">3/7</td>
	</tr>

<tr>
	<td class="file low" data-value="NewDonation.js"><a href="NewDonation.js.html">NewDonation.js</a></td>
	<td data-value="28.57" class="pic low"><div class="chart"><div class="cover-fill" style="width: 28%;"></div><div class="cover-empty" style="width:72%;"></div></div></td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="7" class="abs low">2/7</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="7" class="abs low">2/7</td>
	</tr>

<tr>
	<td class="file low" data-value="NewDonor.js"><a href="NewDonor.js.html">NewDonor.js</a></td>
	<td data-value="19.57" class="pic low"><div class="chart"><div class="cover-fill" style="width: 19%;"></div><div class="cover-empty" style="width:81%;"></div></div></td>
	<td data-value="19.57" class="pct low">19.57%</td>
	<td data-value="46" class="abs low">9/46</td>
	<td data-value="19.57" class="pct low">19.57%</td>
	<td data-value="46" class="abs low">9/46</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="8" class="abs medium">4/8</td>
	<td data-value="19.57" class="pct low">19.57%</td>
	<td data-value="46" class="abs low">9/46</td>
	</tr>

<tr>
	<td class="file low" data-value="NewRecipient.js"><a href="NewRecipient.js.html">NewRecipient.js</a></td>
	<td data-value="25.64" class="pic low"><div class="chart"><div class="cover-fill" style="width: 25%;"></div><div class="cover-empty" style="width:75%;"></div></div></td>
	<td data-value="25.64" class="pct low">25.64%</td>
	<td data-value="39" class="abs low">10/39</td>
	<td data-value="23.68" class="pct low">23.68%</td>
	<td data-value="38" class="abs low">9/38</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="8" class="abs medium">4/8</td>
	<td data-value="25.64" class="pct low">25.64%</td>
	<td data-value="39" class="abs low">10/39</td>
	</tr>

<tr>
	<td class="file high" data-value="PaymentAcknowledge.js"><a href="PaymentAcknowledge.js.html">PaymentAcknowledge.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	</tr>

<tr>
	<td class="file low" data-value="PaymentDetailsForm.js"><a href="PaymentDetailsForm.js.html">PaymentDetailsForm.js</a></td>
	<td data-value="20" class="pic low"><div class="chart"><div class="cover-fill" style="width: 20%;"></div><div class="cover-empty" style="width:80%;"></div></div></td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="10" class="abs low">2/10</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="10" class="abs low">2/10</td>
	</tr>

<tr>
	<td class="file low" data-value="PaymentRequestButton.js"><a href="PaymentRequestButton.js.html">PaymentRequestButton.js</a></td>
	<td data-value="0" class="pic low"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="13" class="abs low">0/13</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="13" class="abs low">0/13</td>
	</tr>

<tr>
	<td class="file medium" data-value="PrivateRoute.js"><a href="PrivateRoute.js.html">PrivateRoute.js</a></td>
	<td data-value="50" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 50%;"></div><div class="cover-empty" style="width:50%;"></div></div></td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	</tr>

<tr>
	<td class="file low" data-value="QRCode.js"><a href="QRCode.js.html">QRCode.js</a></td>
	<td data-value="7.69" class="pic low"><div class="chart"><div class="cover-fill" style="width: 7%;"></div><div class="cover-empty" style="width:93%;"></div></div></td>
	<td data-value="7.69" class="pct low">7.69%</td>
	<td data-value="13" class="abs low">1/13</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="6" class="abs low">0/6</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="6" class="abs low">0/6</td>
	<td data-value="7.69" class="pct low">7.69%</td>
	<td data-value="13" class="abs low">1/13</td>
	</tr>

<tr>
	<td class="file low" data-value="Recipient.js"><a href="Recipient.js.html">Recipient.js</a></td>
	<td data-value="20" class="pic low"><div class="chart"><div class="cover-fill" style="width: 20%;"></div><div class="cover-empty" style="width:80%;"></div></div></td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	</tr>

<tr>
	<td class="file low" data-value="RecipientBalance.js"><a href="RecipientBalance.js.html">RecipientBalance.js</a></td>
	<td data-value="28.57" class="pic low"><div class="chart"><div class="cover-fill" style="width: 28%;"></div><div class="cover-empty" style="width:72%;"></div></div></td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="7" class="abs low">2/7</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	</tr>

<tr>
	<td class="file medium" data-value="RecipientCode.js"><a href="RecipientCode.js.html">RecipientCode.js</a></td>
	<td data-value="66.67" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 66%;"></div><div class="cover-empty" style="width:34%;"></div></div></td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="3" class="abs medium">2/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="1" class="abs low">0/1</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="3" class="abs medium">2/3</td>
	</tr>

<tr>
	<td class="file medium" data-value="RecipientInterests.js"><a href="RecipientInterests.js.html">RecipientInterests.js</a></td>
	<td data-value="66.67" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 66%;"></div><div class="cover-empty" style="width:34%;"></div></div></td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="6" class="abs medium">4/6</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="2" class="abs medium">1/2</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="6" class="abs medium">4/6</td>
	</tr>

<tr>
	<td class="file low" data-value="RecipientProfile.js"><a href="RecipientProfile.js.html">RecipientProfile.js</a></td>
	<td data-value="40" class="pic low"><div class="chart"><div class="cover-fill" style="width: 40%;"></div><div class="cover-empty" style="width:60%;"></div></div></td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	</tr>

<tr>
	<td class="file low" data-value="S3Upload.js"><a href="S3Upload.js.html">S3Upload.js</a></td>
	<td data-value="28.57" class="pic low"><div class="chart"><div class="cover-fill" style="width: 28%;"></div><div class="cover-empty" style="width:72%;"></div></div></td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="21" class="abs low">6/21</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="8" class="abs low">2/8</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="8" class="abs low">2/8</td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="21" class="abs low">6/21</td>
	</tr>

<tr>
	<td class="file low" data-value="ScrollToTop.js"><a href="ScrollToTop.js.html">ScrollToTop.js</a></td>
	<td data-value="0" class="pic low"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	</tr>

<tr>
	<td class="file low" data-value="Tab.js"><a href="Tab.js.html">Tab.js</a></td>
	<td data-value="0" class="pic low"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="9" class="abs low">0/9</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="9" class="abs low">0/9</td>
	</tr>

<tr>
	<td class="file low" data-value="Tabs.js"><a href="Tabs.js.html">Tabs.js</a></td>
	<td data-value="0" class="pic low"><div class="chart"><div class="cover-fill" style="width: 0%;"></div><div class="cover-empty" style="width:100%;"></div></div></td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="11" class="abs low">0/11</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="5" class="abs low">0/5</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Wed Nov 21 2018 10:13:36 GMT+0000 (Greenwich Mean Time)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
