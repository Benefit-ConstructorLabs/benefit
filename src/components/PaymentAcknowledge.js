import React from 'react';
import '../../styles/components/payment-acknowledge.scss'

const PaymentAcknowledge = () => {
  return (
    <section className='acknowledgement' >
      <p className='acknowledgement__notification'>Donation confirmed</p>
      <h2 className='acknowledgement__confirmation'>All set. Thanks for helping "FirstName" out!</h2>
      <h3 className='acknowledgement__question'>Want to save your details for another donation?</h3>
      <button className='btn acknowledgement__account__btn' onClick={event=>event.preventDefault()}>Set up donor account</button>
    </section>
  )
};

export default PaymentAcknowledge;