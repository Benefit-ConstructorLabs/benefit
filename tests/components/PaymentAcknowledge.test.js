import React from 'react';
import { mountWrap, shallowWrap } from './contextWrap';
import { PaymentAcknowledge } from '../../src/components/PaymentAcknowledge';

describe('Component', () => {
  let props;
  let component;
  const wrappedShallow = () => shallowWrap(<PaymentAcknowledge {...props} />);

  // const wrappedMount = () => mountWrap(<PaymentAcknowledge {...props} />);

  beforeEach(() => {
    props = {
      query: {
        refetch: jest.fn(),
      },
    };
    if (component) component.unmount();
  });

  describe('PaymentAcknowledge', () => {
    test.skip('should render correctly', () => {
      const wrapper = wrappedShallow();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
