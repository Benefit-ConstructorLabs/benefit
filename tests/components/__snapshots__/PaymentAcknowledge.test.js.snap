// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component PaymentAcknowledge should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PaymentAcknowledge
    firstName=""
    query={
      Object {
        "refetch": [MockFunction],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="acknowledgement__confirmation"
        >
          All done.Thanks for helping  out!
        </h2>,
        <h3
          className="acknowledgement__account"
        >
          Would you like to set up an account?
        </h3>,
        <Link
          className="btn btn__primary"
          replace={false}
          to="/donor"
        >
          Set up an account
        </Link>,
      ],
      "className": "acknowledgement",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "All done.Thanks for helping  out!",
          "className": "acknowledgement__confirmation",
        },
        "ref": null,
        "rendered": "All done.Thanks for helping  out!",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Would you like to set up an account?",
          "className": "acknowledgement__account",
        },
        "ref": null,
        "rendered": "Would you like to set up an account?",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Set up an account",
          "className": "btn btn__primary",
          "replace": false,
          "to": "/donor",
        },
        "ref": null,
        "rendered": "Set up an account",
        "type": [Function],
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="acknowledgement__confirmation"
          >
            All done.Thanks for helping  out!
          </h2>,
          <h3
            className="acknowledgement__account"
          >
            Would you like to set up an account?
          </h3>,
          <Link
            className="btn btn__primary"
            replace={false}
            to="/donor"
          >
            Set up an account
          </Link>,
        ],
        "className": "acknowledgement",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "All done.Thanks for helping  out!",
            "className": "acknowledgement__confirmation",
          },
          "ref": null,
          "rendered": "All done.Thanks for helping  out!",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Would you like to set up an account?",
            "className": "acknowledgement__account",
          },
          "ref": null,
          "rendered": "Would you like to set up an account?",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Set up an account",
            "className": "btn btn__primary",
            "replace": false,
            "to": "/donor",
          },
          "ref": null,
          "rendered": "Set up an account",
          "type": [Function],
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "childContextTypes": Object {
      "router": [Function],
    },
    "context": Object {
      "router": Object {
        "history": Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        },
        "route": Object {
          "location": Object {},
          "match": Object {},
        },
      },
    },
  },
}
`;
